main.go

```diff
@@ -175,7 +177,7 @@ func run() error {
 		commitMsg = "GitHub Security Advisory"
 	case "glad":
 		gu := glad.NewUpdater()
-		if err := gu.Update(); err != nil {
+		if err := gu.Update(*targetUri, *targetBranch); err != nil {
```

Comment:
I think it is better to pass them to NewUpdater.

Reply:
@knqyf263, good idea, I've moved it, thank you! :bow: